test/acceptance/features/Find_a_Pharmacy.feature

```diff
@@ -0,0 +1,81 @@
+@ui
+@find-a-pharmacy
+@CI2-973
+#noinspection CucumberUndefinedStep
+
```

Comment:
If this is a draft, and there's no steps implemented, should there be a wip tag?

Reply:
Touche!